src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,30 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Todo } from '../../types/todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={`TodoInfo ${completed && 'TodoInfo--completed'}`}
+      key={todo.id}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+
+      <UserInfo user={user} />
```

Comment:
```suggestion
      {user && <UserInfo user={user} />}
```
I think in this case TS will not quarrel, and you don't need to write "user?.email, user?.name"... in UserInfo

Reply:
good point!)